Plugins/Json/include/Acts/Plugins/Json/VolumeBoundsJsonConverter.hpp

```diff
@@ -0,0 +1,51 @@
+// This file is part of the Acts project.
+//
+// Copyright (C) 2022 CERN for the benefit of the Acts project
+//
+// This Source Code Form is subject to the terms of the Mozilla Public
+// License, v. 2.0. If a copy of the MPL was not distributed with this
+// file, You can obtain one at http://mozilla.org/MPL/2.0/.
+
+#pragma once
+
+#include "Acts/Geometry/VolumeBounds.hpp"
+#include "Acts/Plugins/Json/ActsJson.hpp"
+
+#include <array>
+#include <vector>
+
+// Custom Json encoder/decoders. Naming is mandated by nlohmann::json and thus
+// can not match our naming guidelines.
+namespace Acts {
+
+static std::vector<std::string> volumeBoundTypes = {
+    "Cone",     "Cuboid",        "CutoutCylinder",
+    "Cylinder", "GenericCuboid", "Trapezoid"};
+
+void to_json(nlohmann::json& j, const VolumeBounds& bounds);
+
+/// Converstion to surfaceBounds from json
+///
+/// The type is given as a template argument in order to be able
+/// to construct the correct fitting types for surfaces.
+///
+/// @param j the read-in json object
+///
+/// @return a shared_ptr to a surface object for type polymorphism
+template <typename bounds_t>
+std::unique_ptr<bounds_t> volumeBoundsFromJson(const nlohmann::json& j) {
```

Comment:
this could be part of the source file if we do not intend to use it somewhere else